### ISSUE TITLE
Can not run migrations in Django 1.7.8

### ISSUE THREAD

**Issue by antwertinger** [aldryn/aldryn-forms#42]
When i try to run the migrations, the following error occurs:

```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/marc/.virtualenvironments/bug/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/Users/marc/.virtualenvironments/bug/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute
    django.setup()
  File "/Users/marc/.virtualenvironments/bug/lib/python2.7/site-packages/django/__init__.py", line 21, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/marc/.virtualenvironments/bug/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/Users/marc/.virtualenvironments/bug/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/marc/.virtualenvironments/bug/lib/python2.7/site-packages/aldryn_forms/models.py", line 22, in <module>
    User = get_user_model()
  File "/Users/marc/.virtualenvironments/bug/lib/python2.7/site-packages/django/contrib/auth/__init__.py", line 137, in get_user_model
    return django_apps.get_model(settings.AUTH_USER_MODEL)
  File "/Users/marc/.virtualenvironments/bug/lib/python2.7/site-packages/django/apps/registry.py", line 199, in get_model
    self.check_models_ready()
  File "/Users/marc/.virtualenvironments/bug/lib/python2.7/site-packages/django/apps/registry.py", line 131, in check_models_ready
    raise AppRegistryNotReady("Models aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet.
```

It seems that there is a problem with the user model loading during import time in models.py .

I mitigated the problem by removing the call to `get_user_model()` and replacing it with `settings.AUTH_USER_MODEL`

Since I wasn´t sure how to fix this correctly for other versions, I didn´t open a pull request. Let me now if you need more info to fix the issue properly.

**Comment by sephii** [comment]
Same problem here, making the package unusable.

**Comment by mrsarm** [comment]
Same problem with Django 1.8. @antwertinger, can you provide the patch?

**Comment by jumarome** [comment]
Same problem with django 1.7.7 Please help

**Comment by czpython** [comment]
Hello, I'll try to work on 1.7 support in the coming weeks.
As for 1.8, it might take a bit longer.. pull requests always welcome though :)

**Comment by czpython** [comment]
Working version in https://github.com/aldryn/aldryn-forms/pull/54.
Will try to get a release out this week.

**Comment by czpython** [comment]
Fixed by #54
